doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -322,12 +292,16 @@ Requests to the environment
             Search (?x * _ + ?x * _)%Z outside OmegaLemmas.
 
 
-.. cmd:: SearchHead @term
+.. cmd:: SearchHead @one_term {? {| inside | outside } {+ @qualid } }
+
+   Displays the name and type of all hypotheses of the
+   selected goals (if any) and theorems of the current context that have the
+   form :n:`{* P__n -> } (@one_term t__1 ... t__m)`.
```

Comment:
```suggestion
   form :n:`{* P__i -> } (@one_term t__1 ... t__m)`.
```

In fact this can also be `forall` instead of `->` (dependent arrow). Maybe the following would be better:
```suggestion
   form :n:`{? forall {* @binder }, } {* P__i -> } (@one_term t__1 ... t__m)`.
```

Technically, the `{* P__i }` part is redundant with the `forall` part, but it makes for a clearer reading.

Reply:
I guess.  You can have both `forall` and the `P__i?  That falls out from the definition of `->`?  I could also add a sentence explaining how `->` is defined.

Reply:
Indeed, `->` is a specialized `forall` rather than the opposite. This is already explained here BTW: https://coq.github.io/doc/master/refman/language/gallina-specification-language.html#products-forall

Reply:
Added the forall.

Is there a way to display the notation associated with a symbol, like a `Print Notation "->"?  Would be nice if it told you where it's defined.

There should be a way to look up standard notations/nonterminals in the documentation.  A text search on "->" is completely useless.

Reply:
Yes, `Locate` works:

```coq
Locate "->".
(*
Notation
"A -> B" := forall _ : A, B : type_scope (default interpretation)
*)
```

> Would be nice if it told you where it's defined.

That could be an enhancement request. For now, it doesn't.